### ISSUE TITLE
DCD 0.9.0 ?

### ISSUE THREAD

**Issue by BBasile** [dlang-community/discussions#14]
Time to release. What do you think ?

**Comment by wilzbach** [comment]
### Observations
- v0.8.0 was in March 2016
- There have been five alphas for v0.9.0 so far
- Last alpha was in Feb 2017
- There have been [104 commits to master since 0.8.0](https://github.com/dlang-community/DCD/compare/v0.8.0...master) as of now.
- Issues
  - Only two open from 2017
  - I can't spot regressions
- PRs:
  - https://github.com/dlang-community/DCD/pull/366/
- Milestones
  - there are still three issues in the [0.9.0](https://github.com/dlang-community/DCD/milestone/16) milestone

So in summary: I think everything looks good for a 0.9.0 release, but I would wait for @Hackerpilot's OK as he deliberately added these three issues to the 0.9.90 milestone.

Btw I just created a 0.10.0 milestone: https://github.com/dlang-community/DCD/milestones to which these issues could be moved.

**Comment by BBasile** [comment]
This one can be done https://github.com/dlang-community/DCD/issues/319 with very few chances to regress. It's just a recursive file scanner. Actually the most important right now is that it's very stable.
The remaining problems that were planned for 0.9.0 can be reported to 0.9.1.

**Comment by wilzbach** [comment]
@BBasile I just wanted to give @Hackerpilot a couple of days to react, but I think two weeks is more than enough as an objection period.
I have seen that you fixed https://github.com/dlang-community/DCD/issues/319, so from what I can see there's nothing that stands in the way of a DCD 0.9.0 release - happy publishing/tagging!
Maybe even a short announcement at the NG?

**Comment by BBasile** [comment]
Wait a supplemental week. Maybe there two or three simple things that can be done. I think to speed up parsing based on the source size, like for the D-Scanner similar PR.

**Comment by BBasile** [comment]
I wont be able to work _much_ on DCD/D-scanner in the next weeks because of renovations here. So unless Hackerpilot comes and says "I'll close two or three issues soon" i suppose we're good now. In the meantime there's already a beta tag.

**Comment by BBasile** [comment]
And it's very stable right now, it handles more "auto" cases, has a new feature for refact, etc.

**Comment by Hackerpilot** [comment]
I have no problems with a new release, but before we make one the remaining issues in the 0.9.0 milestone should be unassigned from it: https://github.com/dlang-community/DCD/milestone/16

**Comment by BBasile** [comment]
[There's a changelog drafted](https://github.com/dlang-community/DCD/releases/tag/untagged-689428c7412e0673d2d9), ready for potential language fixes.

**Comment by BBasile** [comment]
HackerPilot, you can release. I've prepared a changelog instead of what you did usually. The link above is wrong but if you click [here](https://github.com/dlang-community/DCD/releases) you shoud see it.

**Comment by BBasile** [comment]
ok done. thanks for your agreement HP, it's very stable now and people building with DUB without specifying the version won't have any bad surprise.
